pipeline/leech/main.go

```diff
@@ -139,6 +149,38 @@ func realMain(ctx context.Context) error {
 	return nil
 }
 
+func splitAndValidate(dottedTableName string) error {
+	parts := strings.Split(dottedTableName, ".")
```

Comment:
```suggestion
	parts := strings.SplitN(dottedTableName, ".", 2)
```

Reply:
I'm not sure I like that, as it shadows any invalid input.

For example datasetID.tablename.oops would be split into datasetID, tablename.oops with no error

Reply:
That's fair.